docs/libcudacxx/extended_api/memory_access_properties/access_property.rst

```diff
@@ -1,22 +1,11 @@
 .. _libcudacxx-extended-api-memory-access-properties-access-property:
 
-cuda::access_property
+``cuda::access_property``
 =========================
 
-Defined in header ``<cuda/annotated_ptr>``:
+Defined in header ``<cuda/annotated_ptr>``.
 
-.. code:: cuda
-
-   namespace cuda {
-   class access_property;
-   } // namespace cuda
-
-The class ``cuda::access_property`` is a `LiteralType <https://en.cppreference.com/w/cpp/named_req/LiteralType>`_
-that provides an opaque encoding for properties of memory operations. It is used in combination with
-:ref:`cuda::annotated_ptr <libcudacxx-extended-api-memory-access-properties-annotated-ptr>`,
-:ref:`cuda::associate_access_property <libcudacxx-extended-api-memory-access-properties-associate-access-property>` and
-:ref:`cuda::apply_access_property <libcudacxx-extended-api-memory-access-properties-apply-access-property>`
-to *request* the application of properties to memory operations.
+The class ``cuda::access_property`` provides an opaque encoding for *L2 cache memory residence* control and *memory space* properties. It is used in combination with :ref:`cuda::annotated_ptr <libcudacxx-extended-api-memory-access-properties-annotated-ptr>`, :ref:`cuda::associate_access_property <libcudacxx-extended-api-memory-access-properties-associate-access-property>` and :ref:`cuda::apply_access_property <libcudacxx-extended-api-memory-access-properties-apply-access-property>` to *request* the application of properties to memory operations.
```

Comment:
As a first time reader I would appreciate a link: [L2 cache memory residence](https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html?highlight=l2%2520residency#device-memory-l2-access-management).

I make this comment with the purpose to link it in #4503